Postfix _Evaluation.c

```diff
@@ -0,0 +1,84 @@
+//Postfix Evaluation
+#include<stdio.h>
+#include<string.h>
+#include<ctype.h>
+struct stack
+{
+   char A[100];
+   int Top;
+};
+void push(struct stack *ps,char x)
+{
+    if(ps->Top==99)
+    printf("\nStack is Full\n");
+    else
+    {
+      ps->Top=ps->Top+1;
+      ps->A[ps->Top]=x;
+    }
+}
+
+int evaluate(int op1,int op2,char c)
+{
+   if(c=='+')
```

Comment:
Spaces around operators like `==`, `<`, `=` etc. Would be great and improves readability.